### PR TITLE
Python EventHubs test code revamp

### PR DESCRIPTION
**Pull request by YijunXieMS**
This PR has only test code.
Added offline unittest for EventData as a starter
Put all live test under folder "livetest"

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by YijunXieMS** [comment]
Wrong PR. Should be merge to eventhubs_preview4 instead of master

File: sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_receive_async.py [hunk 1]
```diff
@@ -325,7 +325,9 @@ def batched():
 @pytest.mark.asyncio
 @pytest.mark.liveTest
 async def test_receive_run_time_metric_async(connstr_senders):
-    pytest.skip("Disabled for uamqp 1.2.2. Will enable after uamqp 1.2.3 is released.")
+    from uamqp import __version__ as uamqp_version
+    if uamqp_version < "1.2.3":
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_receive_async.py]: not sure about whether the comparison between version string would work well in all case, but we could use the python built-in module:

from distutils.version import StrictVersion
if StrictVersion(uamqp_version) < StrictVersion('1.2.3'):

File: sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py [hunk 2]
```diff
@@ -282,7 +282,9 @@ def test_receive_over_websocket_sync(connstr_senders):
 
 @pytest.mark.liveTest
 def test_receive_run_time_metric(connstr_senders):
-    pytest.skip("Disabled for uamqp 1.2.2. Will enable after uamqp 1.2.3 is released.")
+    from uamqp import __version__ as uamqp_version
+    if uamqp_version < "1.2.3":
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py]: same to the above version string comparsion

File: sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_receive_async.py (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_receive_async.py]: Good point. Changed

File: sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py]: Changed

**Pull request merged**